### PR TITLE
feat(route): Isabelle dev blog

### PR DESCRIPTION
**Pull request by Ritsuka314**
## Example for the Proposed Route(s) / 路由地址示例

```routes
/sketis/isabelle-dev/blog/1
/sketis/isabelle-dev/blog/2
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
- [ ] Full text / 全文获取
- [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/sketis/isabelle-dev/blogs/namespace.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export const namespace: Namespace = {
+    name: 'isabelle-dev',
+    url: 'https://isabelle-dev.sketis.net/phame/',
+    description: `
+:::tip
+Isabelle/Phabricator Blogs
+
+-   Isabelle Release: \`https://isabelle-dev.sketis.net/phame/blog/view/2/\`
+-   Isabelle News: \`https://isabelle-dev.sketis.net/phame/blog/view/1/\`
+:::`,
```
Comment (TonyRL) [inline lib/routes/sketis/isabelle-dev/blogs/namespace.ts:12]: These should descript the namespace
```suggestion
    name: 'Sketis | Website of Dr. Makarius Wenzel',
    url: 'sketis.net',
```

File: lib/routes/sketis/isabelle-dev/blogs/blogs.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            .map((item_) => {
+                const item = $(item_);
+                const title = item.find('.remarkup-header').first();
+                const subtitle = item.find('.phui-document-summary-subtitle').first();
+                const date = subtitle.find('strong').first()[0].nextSibling.data.slice(4); // parse starts after ' on '
+                return {
+                    title: title.text(),
+                    // We need an absolute URL for `link`, but `a.attr('href')` returns a relative URL.
+                    link: `${baseUrl}${title.find('a').attr('href')}`,
+                    description: item.find('.phui-document-summary-body').text(),
```
Comment (TonyRL) [inline lib/routes/sketis/isabelle-dev/blogs/blogs.ts:58]:
```suggestion
                    description: item.find('.phui-document-summary-body').html(),
```

File: lib/routes/sketis/isabelle-dev/blogs/blogs.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    'isabelle-dev.sketis.net/phame/blog/',
+    'isabelle-dev.sketis.net/phame/blog/view/:blog/',
+    'isabelle-dev.sketis.net/phame/post/',
+    'isabelle-dev.sketis.net/phame/post/view/:post_id/:post_title/',
+];
+
+export const route: Route = {
+    path: '/isabelle-dev/blog/:blog',
+    categories: ['programming'],
+    example: '/sketis/isabelle-dev/blog/{1,2}',
```
[the remark this answers is not in the document]
Comment (Ritsuka314) [reply lib/routes/sketis/isabelle-dev/blogs/blogs.ts:17]: done

File: lib/routes/sketis/isabelle-dev/blogs/namespace.ts (hunk 1 above)
Comment (Ritsuka314) [reply lib/routes/sketis/isabelle-dev/blogs/namespace.ts:12]: I'm changing the namespace to `isabelle-dev`, which is more approperate for this situation.

Comment (TonyRL) [reply lib/routes/sketis/isabelle-dev/blogs/namespace.ts]: You can use RouteItem.description for route specific description.

> changing the namespace to `isabelle-dev`

In principle, the namespace should be **the same** as the secondary domain of the main website where you are making the RSS feed. For example, if you are making an RSS feed for https://github.com/DIYgod/RSSHub/issues, the secondary domain is `github`.

docs: https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace

**Pull request merged**